src/components/Image/Image.tsx

```diff
@@ -8,6 +8,16 @@ import { ImageBehavior } from '../../lib/accessibility'
 
 /**
  * An image is a graphic representation of something.
+ * @accessibility
+ * Default behavior: ImageBehavior
+ *  - attribute "aria-hidden='true'" is applied on img tag, if there is no 'alt' attribute provided
+ *
+ * If image should be visible to screen readers, then it has to have 'alt' attribute specified.
+ *
+ * Other considerations:
+ *  - when alt attribute is empty, then Narrator in scan mode navigate to image and narrate it as empty paragraph
+ *  - when image has role='presentation' then screen readers navigate to the element in scan/virtual mode. To avoid it the attribute "aria-hidden='true'" should be used
+ *  - when image is visible for screen reader and contains another attribute as aria-label/arialabbeledby/title, then verify different screen readers narrations
```

Comment:
why should the users test/verify this? let's try to provide more concrete information here